install-dev/upgrade/sql/1.7.7.0.sql

```diff
@@ -800,3 +800,14 @@ VALUES (NULL, 'actionOrderMessageFormBuilderModifier', 'Modify order message ide
        (NULL, 'actionCreditSlipGridPresenterModifier', 'Modify credit slip grid template data',
         'This hook allows to modify data which is about to be used in template for credit slip grid', '1')
 ;
+
+INSERT IGNORE INTO `ps_authorization_role` (`slug`) VALUES
+  ('ROLE_MOD_TAB_DEFAULT_CREATE'),
+  ('ROLE_MOD_TAB_DEFAULT_READ'),
+  ('ROLE_MOD_TAB_DEFAULT_UPDATE'),
+  ('ROLE_MOD_TAB_DEFAULT_DELETE');
+
+INSERT IGNORE INTO `ps_access` (`id_profile`, `id_tab`, `view`, `add`, `edit`, `delete`) VALUES ('1', (SELECT `id_tab` FROM `ps_tab` t WHERE t.class_name = 'AdminTAbs' LIMIT 1), '1', '1', '1', '1');
+
+INSERT IGNORE INTO `ps_tab_lang` (`id_lang`, `id_tab`, `name`) VALUES
+	(1, (SELECT `id_tab` FROM `ps_tab` WHERE `class_name` = 'AdminTabs'), 'Menus');
```

Comment:
Use `PREFIX_` instead of `ps_`

Reply:
Well done ;)

Reply:
I'm getting older, but I still have my eyes 🤠